core/controllers/acl_decorators.py

```diff
@@ -1846,9 +1846,11 @@ def test_can_edit_skill(self, skill_id, **kwargs):
         if not self.user_id:
             raise base.UserFacingExceptions.NotLoggedInException
 
-        skill_rights = skill_services.get_skill_rights(skill_id, strict=False)
-        if skill_rights is None:
-            raise base.UserFacingExceptions.PageNotFoundException
+        for single_skill_id in skill_id.split(','):
```

Comment:
I think you should rename skill_id to comma_separated_skill_ids or similar -- always make sure variable names semantically capture the meaning of the underlying "thing". (Then you can change single_skill_id to skill_id.)

For bonus points, validate that each skill_id is a "valid-looking" skill_id (alphanumeric with at most X chars, or whatever the charset is) and raise 400 error if not. In general when writing code at the controller layer, assume that anything at all can be passed in.

Reply:
Done.

Reply:
Is this done?